Resources/Prototypes/Loadouts/role_loadouts.yml

```
@@ -106,6 +106,7 @@
   - Glasses
   - Survival
   - Trinkets
+  - LawyerStamps # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)